dv/uvm/core_ibex/waivers/coverage_waivers_xlm.tcl

```diff
@@ -0,0 +1,25 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Exclude standard primitives that are verified elsewhere
+exclude -type "prim_secded_*"
+exclude -type "prim_lfsr"
+exclude -type "prim_flop"
+exclude -type "prim_prince"
+exclude -type "prim_subst_perm"
+exclude -type "prim_ram_1p*"
+
+# Exclude code coverage from aux code used for gathering functional coverage
+exclude -type "core_ibex_fcov_if" -metrics code:statement:fsm:assertion
+exclude -type "core_ibex_pmp_fcov_if" -metrics code:statement:fsm:assertion
+
+# RVFI signals not present in real design so not relevant to coverage closure
+exclude -type "ibex_top" -toggle "rvfi*"
+# hart_id_i and boot_addr_i are intended to be hard-wired inputs that do not
+# toggle
+exclude -type "ibex_top" -toggle "hart_id_i"
+exclude -type "ibex_top" -toggle "boot_addr_i"
```

Comment:
I feel like these can be covered in V3 tests and should not be excluded since they are design ports. as opposed to parameters.

Reply:
The issue is they probably should be parameters and they're intended to be totally static. As it happens I think you can alter them at will and get some sensible behaviour but they are meant to be wired to a constant.

At this stage I don't want to go altering Ibex's parameter set so I think they should remain design ports but I think it's reasonable not to test toggling, especially for the OT use case where they are wired to a constant in the earlgrey top-level.